### ISSUE TITLE
Chasser une fonction !

### ISSUE THREAD

**Issue by nborie** [PremierLangage/Yggdrasil#47]
**Titre de l'exercice :** Chasser une fonction !
**Public visé :** Tout étudiant en fin de lycée ou dans le supérieur ayant suivi des cours d'analyse de fonctions
**Contexte d’utilisation :** Possiblement en autonomie
**Objectif pédagogique :** Travailler la mémoire et l'imagination avec ses connaissances sur les fonctions
**TAGS :** mathématique, analyse de fonction
**Description de l'exercice (Fichier) :** On tire au sort des conditions sur des fonctions (exemple : f(1) = -2, f'(0) = 0, f est 2-Pi périodique, etc...). Pas trop de conditions, ni trop peu...
**Champs de saisie :** L'éléve répond par une fonction d'une variable et réelle
**Valeurs aléatoires et algorithme qui les génère :** déterminé par le codeur
**Comment détermine-t-on si la réponse est juste :** déterminé par le codeur (simpy???)
**Feedback à l'étudiant :** On lui liste les conditions vérifiées et celles non vérifiées
**Fichier de données (tableau):** 
**Description de l'exercice (texte):** Trouve une fonction qui vérfie les conditions suivante !

**Comment by gimau** [comment]
Voici une première version de cet exercice.
https://pl.u-pem.fr/filebrowser/option?name=test_pl&path=Yggdrasil/conceptexo/Mathematics/exercises/algebra/polynomials/function_hunter.pl

**Comment by jcdufourd** [comment]
Je vois ceci dans mon chrome/mac
Ecrire une fonction $% f(x) %$ de période $% 7 %$ telle que : $% f(0) = -3%$ et $% f'(0) = - \frac{10 \pi}{7}%$.
Qu'est-ce qui me manque ? Une extension ? Un setting ?

**Comment by nborie** [comment]
Les $% code en latex %$ ne sont pas toujours formatés correctement (essaie d'actualiser plein de fois d'affilé et paf, parfois ça marche...).

Il s'agit d'un bug identifié de playexo.py dans le serveur Premier Langage...

**Comment by qcoumes** [comment]
Oui, le bug est identifié et corrigé sur la version en cours qui sera déployé prochainement.
